### ISSUE TITLE
A11y_Playwright.dev_ScreenReader: Narrator is announcing as graphic for the "Caution" present in "Type characters" section.

### ISSUE THREAD

**Issue by Raghu-addepalli** [microsoft/playwright.dev#1268]
### Actual Result:
Narrator is announcing grahics for "Caution" image.

**Note: Issue is not reproducing with NVDA.**
Note: Similar issue is also observed in below links where the similar type of controls are available.

https://playwright.dev/docs/test-assertions
https://playwright.dev/docs/running-tests
https://playwright.dev/docs/locators
https://playwright.dev/docs/api/class-locator
https://playwright.dev/docs/writing-tests
https://playwright.dev/docs/api/class-playwright
https://playwright.dev/docs/intro
https://playwright.dev/docs/api/class-page

### Expected Result:
Narrator should announce announce "Caution graphic" for an  "Caution" image.

### GitHub Tags:
#A11yMAS;#A11yTCS;#Playwright releases;#P1_Playwright.dev_Web_Jan2024;#A11ySev3;#AI Limited;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG1.3.1;#Info and Relationships;#Narrator;#Win11;#Element:Graphic#P1_WCP

### Environmental Details:
App Name: Playwright.dev
#URL: https://playwright.dev/docs/input

### OS & Browser Details:
Microsoft Windows 11 Enterprise
Version 22621.3007
Microsoft Edge
Version 120.0.2210.144 (Official build) (64-bit)
Chrome Version: Version 120.0.6099.225 (Official Build) (64-bit)
NVDA Version: 2023.3

### Repro Steps:

1. Hit the URL: https://playwright.dev/docs/input.
2. Turn on Narrator in Scan mode.
3. TAB to "Caution" present in "Type characters" section.
4. Now Check that Narrator is announcing as graphic for the "Caution" or not.

### User Impact:
Screen Reader users will not get about the "Caution" if no information is announced by Narrator.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Attachment:
[Screen Reader attachments.zip](https://github.com/microsoft/playwright.dev/files/14093812/Screen.Reader.attachments.zip)
